### PR TITLE
update lollipop when switch transcript

### PR DESCRIPTION
**Pull request by leexgh**
Fix: https://github.com/genome-nexus/genome-nexus/issues/513

Should update protein change when switching transcripts

### CODE REVIEW

File: src/util/variantUtils.ts [hunk 1]
```diff
@@ -38,5 +40,5 @@ export function variantToMutation(
 
 export function getProteinPosStart(proteinChange: string | undefined) {
     const proteinPosition = getProteinPositionFromProteinChange(proteinChange);
-    return proteinPosition ? proteinPosition.start : 0;
+    return proteinPosition ? proteinPosition.start : -1;
```
Comment (leexgh) [inline src/util/variantUtils.ts:43]: set to `-1`, otherwise when there is no protein position it will show lollipop at `0` postion

Comment (inodb) [reply src/util/variantUtils.ts:43]: i guess this was correct for TERT promotor: https://deploy-preview-89--genome-nexus-frontend.netlify.app/variant/5:g.1295228G%3EA

Comment (inodb) [reply src/util/variantUtils.ts:45]: not sure exactly how we would separate that case tho

Comment (inodb) [reply src/util/variantUtils.ts:45]: maybe based on `c.x` position or something

Comment (inodb) [reply src/util/variantUtils.ts:45]: maybe ok to leave as -1 for now

**Pull request merged**